File: ex_004.py
array = [1, 4, 8, 25, 1101, 1400, 2100, 4001]

def linear_search(array, search_element):
    for i in range(len(array)):
        if array[i] == search_element:
            return True
    return False

print(linear_search(array, 1400))
    

File: ex_004.js
function search(array, search_element) {
  for (let i = 0; i < array.length; i++) {
    if (array[i] === search_element) {
      return true;
    }
  }
  return false;
}

array = [1, 4, 8, 25, 1101, 1400, 2100, 4001];

console.log(search(array, 1400));
